src/containers/AddCheckInOut/index.tsx

```diff
@@ -67,22 +71,24 @@ const AddCheckInOut = () => {
 
   return (
     <Box>
-      <FormControlLabel
-        sx={{
-          display: 'block',
-          alignContent: 'center',
-          // color: status === 'Checked-in' ? 'green' : 'white',
-        }}
-        label={status}
-        control={
-          <Switch
-            checked={switchState}
-            onChange={handleSwitchChange}
-            name="checkin"
-            color="success"
-          />
-        }
-      />
+      {isEmployee && (
```

Comment:
I see that this AddCheckInOut component is used only for employee. So instead of checking isEmployee in this component, should we have checked it in parent component and do not render the component if the role is not employee.

Reply:
Yah i have done that but i just wanted to double check them.
I have done in the navbar that if the role is employee then only AddCheckInOut component could be accessed.